scripts/hatvaults-deploy.js

```diff
@@ -0,0 +1,115 @@
+const ADDRESSES = require("./addresses.json");
+const fs = require("fs");
+const { getImplementationAddress } = require('@openzeppelin/upgrades-core');
+
+async function main(
+  rewardsToken = ADDRESSES[network.name].hatToken,
+  startBlock = null,
+  rewardPerEpoch = [
+    "44130000000000000000000",
+    "44130000000000000000000",
+    "88250000000000000000000",
+    "77880000000000000000000",
+    "68730000000000000000000",
+    "60650000000000000000000",
+    "53530000000000000000000",
+    "47240000000000000000000",
+    "41690000000000000000000",
+    "36790000000000000000000",
+    "32470000000000000000000",
+    "28650000000000000000000",
+    "25280000000000000000000",
+    "22310000000000000000000",
+    "19690000000000000000000",
+    "17380000000000000000000",
+    "15340000000000000000000",
+    "13530000000000000000000",
+    "11940000000000000000000",
+    "10540000000000000000000",
+    "9300000000000000000000",
+    "8210000000000000000000",
+    "7240000000000000000000",
+    "6390000000000000000000",
+  ],
+  epochLength = "195200",
+  governance = ADDRESSES[network.name].governance,
+  swapToken = ADDRESSES[network.name].hatToken,
+  whitelistedRouters = ADDRESSES[network.name].whitelistedRouters,
+  tokenLockFactory = ADDRESSES[network.name].tokenLockFactory,
+  silent = false
```

Comment:
why only hat vaults has silent option?

Reply:
The silent is only good for using the script in tests and only that script is used in the tests currently